src/databricksbundle/widgets/DatabricksWidgetsTest.py

```diff
@@ -124,6 +158,23 @@ def test_multiselect_single_value_invalid(self):
             str(error.exception),
         )
 
+    def test_multiselect_label_wrong_name(self):
+        with self.assertRaises(Exception) as error:
+            self.__widgets.add_multiselect("mymulti~", ["January", "February", "March"], ["January"])
+
+        self.assertEqual(
+            "The name you provided is incorrect, please provide name containing only alpha-numeric letters and _",
+            str(error.exception),
+        )
+
+    def test_multiselect_label_generation(self):
```

Comment:
I do not see any multiselects in the test, only text fields

Reply:
fixed